code-challenges.js

```diff
@@ -28,18 +30,36 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// Pseudo code:
+// Pseudo code: First we would make a new const and call it CombinedRuns or WorldSeriesRuns
+// const CombinedRuns = [2, 5, 2, 2, 4, 6, 3, 5, 3]
+// const WorldSeriesRuns = [2, 5, 2, 2, 4, 6, 3, 5, 3]
+// Since we know the expected output is 9 we know they want the arrays to show all in one
+// console.log(CombinedRuns)
+// console.log(WorldSeriesRuns)
+// Either one should return 9
+// Can also use console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
```

Comment:
console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns) is a great way to solve it.  Otherwise you are doing the work of the computer.  :)